appveyor.yml

```diff
@@ -1,48 +1,43 @@
 version: '{build}-{branch}'
 
+image: Visual Studio 2019
+
 branches:
-  only:
-    - master
+  except:
+    - playground
 
 clone_depth: 1
 
-
 init:
 - ps: |
-    function craft($target) {
-        & C:\Python36\python.exe "C:\CraftMaster\CraftMaster\CraftMaster.py" --config "$env:APPVEYOR_BUILD_FOLDER\appveyor.ini" --variables "APPVEYOR_BUILD_FOLDER=$env:APPVEYOR_BUILD_FOLDER" --target $target -c $args
+    function craft() {
+        cmd /C "echo %PATH%"
+        & "C:\Python39-x64\python.exe" "C:\CraftMaster\CraftMaster\CraftMaster.py" --config "$env:APPVEYOR_BUILD_FOLDER\appveyor.ini" --variables "APPVEYOR_BUILD_FOLDER=$env:APPVEYOR_BUILD_FOLDER" --target $env:TARGET -c $args
```

Comment:
@mgallien Are you sure there is a folder named `Python39-x64`? Usually, `-x64` is not added to the folder name.

Reply:
see my other reply about the fact that the path are determined by Appveyor CI image